x-pack/legacy/plugins/ml/server/models/data_recognizer/modules/siem_auditbeat_ecs/ml/suspicious_login_activity_ecs.json

```diff
@@ -0,0 +1,36 @@
+{
+  "job_type": "anomaly_detector",
+  "description": "SIEM Auditbeat: Detect unusually high increases in authentication attempts (beta)",
+  "groups": ["siem"],
+  "analysis_config": {
+    "bucket_span": "15m",
+    "detectors": [
+      {
+        "detector_description": "high rate of authentication attempts",
```

Comment:
"high number of authentication attempts"

Reply:
Done in d33a13304d9b8cb3b6555dc7b29711b7319b7f57